Dockerfile

```diff
@@ -21,6 +21,7 @@ ARG ARCH
 ADD downloads/kubectl-linux-${ARCH} /usr/local/bin/kubectl
 ADD downloads/helm-linux-${ARCH} /usr/local/bin/helm
 ADD downloads/oc-linux-${ARCH} /usr/local/bin/oc
+ADD downloads/subctl-linux-${ARCH} /usr/local/bin/subctl
```

Comment:
Does this need a downstream change? Just checking.

Reply:
Yes, it will require different change in the downstream Dockerfile.in to pull the CLI binary from a downstream source.  That will come in a PR for the downstream repo.